src/services/sqlProjectsService.ts

```diff
@@ -415,4 +415,55 @@ export class SqlProjectsService implements mssql.ISqlProjectsService {
 		const params: mssql.SqlProjectParams = { projectUri: projectUri };
 		return this._client.sendRequest(contracts.GetSqlObjectScriptsRequest.type, params);
 	}
+
+	/**
+	 * Add a none script to a project
+	 * @param projectUri Absolute path of the project, including .sqlproj
+	 * @param path Path of the script, including .sql and .publish.xml, relative to the .sqlproj
+	 */
+	 public async addNoneItem(projectUri: string, path: string): Promise<mssql.ResultStatus> {
```

Comment:
do these also need to be added in vscode-mssql.d.ts? I don't see them there in the [vscode-mssql](https://github.com/microsoft/vscode-mssql/blob/main/typings/vscode-mssql.d.ts) version of the file